core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -366,7 +381,7 @@ var ExplorationEditorPage = function() {
   this.navigateToMainTab = async function() {
     await action.waitForAutosave();
     await action.click('Main tab button', navigateToMainTabButton);
-    await action.click('Neutral element', neutralElement);
+    await action.click('Neutral element', neutralElement, true);
```

Comment:
Why are we clicking a masked element here? It doesn't seem like a migration should have changed whether an element is masked

Reply:
removed